### PR TITLE
Add BookPlayer by @GianniCarlo

### PR DESCRIPTION
**Pull request by GianniCarlo**
1. [x] Project URL: https://github.com/TortugaPower/BookPlayer
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Avoid iOS or open-source in description as it is assumed
5. [x] Use this commit title format if applicable: Add app-name by @github-username
6. [x] Use approved format for your entry

### CODE REVIEW

**Comment by GianniCarlo** [comment]
Those link issues aren't related to the app I'm trying to add 🤔

File: contents.json [hunk 1]
```diff
@@ -16641,6 +16641,27 @@
...
       "suggested_by": "@RMizin"
+    },
+    {
+      "title": "BookPlayer",
+      "category-ids": [
+        "media"
+      ],
+      "description": "Player for your DRM-free audiobooks",
+      "source": "https://github.com/TortugaPower/BookPlayer",
+      "itunes": "https://itunes.apple.com/us/app/bookplayer-audio-book-player/id1138219998",
```
Comment (vpeschenkov) [inline contents.json]: Could you remove 'us' from the itunes link?

**Comment by vpeschenkov** [comment]
Hi @GianniCarlo, thank you for your contribution! Could you fix the link?

--
> Those link issues aren't related to the app I'm trying to add 🤔


Don't worry about them.

File: contents.json (hunk 1 above)
Comment (GianniCarlo) [reply contents.json]: sure 👍

**Comment by GianniCarlo** [comment]
@vpeschenkov updated the link without the 'us' 👌

**Pull request merged**